neofs-testlib/neofs_testlib/env/templates/neofs_env_config.yaml

```diff
@@ -46,8 +46,8 @@ binaries:
 
     neofs_contract:
         repo: 'nspcc-dev/neofs-contract'
-        version: 'v0.21.0'
-        file: 'neofs-contract-v0.21.0.tar.gz'
+        version: 'v0.22.0'
```

Comment:
could you remind me why do we need it here?

Reply:
Tests with main chain need neofs contract to be deployed.